source/new-ia/by-role/system-admin/kubernetes/studio-clustering-with-kubernetes-deployment.rst

```diff
@@ -103,7 +103,7 @@ You can easily access Studio through the Authoring load balancer. To get the loa
 Setup Delivery (optional)
 -------------------------
 
-You can use the Delivery Simple example under https://github.com/craftercms/kubernetes-deployments/tree/develop/delivery/simple with this Authoring cluster example:
+You can use the Delivery Simple example under https://github.com/craftercms/kubernetes-deployments/tree/master/delivery/simple with this Authoring cluster example:
```

Comment:
Are you sure of the branch?

Reply:
Yes, we should use master to avoid having to changing the docs when develop becomes something else.